src/rmarkdown/preview.ts

```diff
@@ -86,10 +86,16 @@ class RMarkdownPreview extends vscode.Disposable {
         this.htmlLightContent = $.html();
 
         // make the output chunks a little lighter to stand out
-        const chunkCol = String(config().get('rmarkdown.chunkBackgroundColor'));
-        const colReg = /[0-9.]+/g;
-        const regOut = chunkCol.match(colReg);
-        const outCol = `rgba(${regOut[0] ?? 100}, ${regOut[1] ?? 100}, ${regOut[2] ?? 100}, ${Number(regOut[3]) + 0.05 ?? .5})`;
+        let chunkCol = String(config().get('rmarkdown.chunkBackgroundColor'));
+        let outCol: string;
+        if (chunkCol) {
+            const colReg = /[0-9.]+/g;
+            const regOut = chunkCol.match(colReg);
+            outCol = `rgba(${regOut[0] ?? 100}, ${regOut[1] ?? 100}, ${regOut[2] ?? 100}, ${Number(regOut[3]) + 0.05 ?? .5})`;
+        } else {
+            chunkCol = 'var(--vscode-selection-background)';
+            outCol = 'var(--vscode-selection-background)';
```

Comment:
Should we make the chunk and output appear slightly different?

<img width="491" alt="image" src="https://user-images.githubusercontent.com/4662568/142085455-ca952a3f-b292-4558-ba50-add420eb8cff.png">

<img width="489" alt="image" src="https://user-images.githubusercontent.com/4662568/142085479-55cdda2c-091d-4e4b-9d3f-69652cdb5df7.png">

Reply:
Looks like we should use something like `rgba(128, 128, 128, 0.1)` and `rgba(128, 128, 128, 0.05)` to get a more consistent coloring under different themes.